src/hooks/eventLogger.ts

```diff
@@ -9,13 +9,20 @@ interface EventArguments {
 
 let eventLogger: EventLogger
 
-function getEventLogger(): EventLogger {
+export function getEventLogger(): EventLogger {
     if (!eventLogger) {
         eventLogger = new EventLogger('https://sourcegraph.com')
     }
     return eventLogger
 }
 
+export const logCodeSnippetCopied = (snippetName: string = ''): void => {
+    const eventArguments = { snippetName }
+
+    // eslint-disable-next-line @typescript-eslint/no-floating-promises
+    getEventLogger()?.log('CodeSnippetCopied', eventArguments, eventArguments)
```

Comment:
```suggestion
    getEventLogger().log('CodeSnippetCopied', eventArguments, eventArguments)
```

I don't think this can be null (the signature of `getEventLogger()` is also not nullable)

Reply:
You're right, I wrote this because the other `getEventLogger` calls used `.?` but it looks like it's unnecessary